testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -198,6 +198,17 @@ def test_start_stop_step_parallel(u, run_kwargs, frames, client_FrameAnalysis):
     assert_almost_equal(an.times, frames+1, decimal=4, err_msg=TIMES_ERR)
 
 
+def test_reset_n_parts_to_n_frames(u):
+    # Issue #4685
+    a = FrameAnalysis(u.trajectory)
+    with pytest.warns(UserWarning, match='Set to'):
```

Comment:
```suggestion
    with pytest.warns(UserWarning, match="Set 'n_parts' to"):
```
The change suggested above makes the test also a bit more specific.